### DISCUSSION TITLE
How should I merge a big number of images?

### DISCUSSION THREAD

**Discussion by tene38** [cat-milk/Anime-Girls-Holding-Programming-Books#382] [General]
Hey, I found my old collection, and identified about ~140 images which I could merge. How should I go about it? Should I submit it in a single, giant, pull request, or should I break it into smaller ones, based on some rule (eg. book type)?

**Comment by Asday** [comment]
Thinking about why a PR to this repo would be accepted or rejected, it's going to be about the quality of the image/images included right?  Ideally they'd all be their own PR so they could be individually rejected but that'd be a pain in the neck to review.  You could submit them all in one mega PR, but then github's web interface takes a dump and it becomes a pain in the neck again, especially if some need to be rejected and you have to add commits to the PR to remove them.

I think the best way might be for you to categorise them according to quality (and appropriateness for the repo - that means no boys), and submit a couple PRs.  One for the stuff you're sure is top qual, one for the stuff you're not sure about, maybe another for the stuff you don't think is but other people might like, and so forth.

**Reply by tene38** [reply]
Categorizing by quality seems to be a good idea. I'll also split content which i'm not sure will fit into separate pull requests. Thanks.

Now, naming images will be a pain.

**Reply by Asday** [reply]
Pain it might be, but I'm sure your efforts will be appreciated.
